nethttp/server.go

```diff
@@ -68,12 +59,11 @@ func Middleware(tr opentracing.Tracer, h http.Handler, options ...MWOption) http
 		ext.HTTPMethod.Set(sp, r.Method)
 		ext.HTTPUrl.Set(sp, r.URL.String())
 		ext.Component.Set(sp, "net/http")
-		w = &statusCodeTracker{w, 200}
 		r = r.WithContext(opentracing.ContextWithSpan(r.Context(), sp))
 
-		h.ServeHTTP(w, r)
+		m := httpsnoop.CaptureMetrics(h, w, r)
```

Comment:
There is a benchmark in httpsnoop that seems to imply an overhead of about 500ns: https://github.com/felixge/httpsnoop#performance